### DISCUSSION TITLE
SSID (RSSI) showed data questeion

### DISCUSSION THREAD

**Discussion by sandy1981Nett** [arendst/Tasmota#13039] [Q&A] [answered]
hello togehter, 

i have problems with my Wlan  reception , on the Website from the Tasmota devieces i can see the RSSI in dbm, from mqtt tasmota send the % off rssi .... how is this value calculated in %? i Like too see in my Visulation a card off the rssi from the Tasmonta Devices in dgm ( Shelly ) can i colculated the rssi in dbm from the rssi in % ? 

thank you

Christian from Germany (Nettetal)

**Comment by sfromis** [comment, marked as answer]
If you want the "classic" RSSI value, it is already there in the field "Signal", no calculation needed.

The Tasmota RSSI is calculated as:  rssi% = 2 * (rssi:dBm + 100) for values bracketed to a range from -100 to -50.

**Reply by sandy1981Nett** [reply]
thank you, can i send the classic RSSI by MQTT to my Broker ?

**Reply by barbudor** [reply]
As said above it is already provided in the STATE message that is sent to your broker:
`{"Time":"2021-09-02T11:29:58+02:00",...., Wifi":{... "RSSI":44,"Signal":-78...}}`

**Reply by sandy1981Nett** [reply]
thank you i edit my Broker on the IoBroker to full mqtt State now i see it.
